lib/grizzly/zwave/commands/switch_multilevel_set.ex

```diff
@@ -5,20 +5,17 @@ defmodule Grizzly.ZWave.Commands.SwitchMultilevelSet do
   Params:
 
     * `:target_value` - '`:off`, `:previous` or a value between 0 and 99
-    * `:duration` - How long the switch should take to reach target value,
-      * 0 -> instantly
-      * 1..127 -> seconds
-      * 128..254 -> minutes + 127
-      * 255 -> factory default (option v2)
-
+    * `:duration` - How long in seconds the switch should take to reach target value or the factory default (:default)
+                    Beyond 127 seconds, the duration is truncated to the minute. E.g. 179s is 2 minutes and 180s is 3 minutes
+                    (optional v2)
   """
 
   @behaviour Grizzly.ZWave.Command
 
   alias Grizzly.ZWave.{Command, DecodeError}
   alias Grizzly.ZWave.CommandClasses.SwitchMultilevel
 
-  @type param :: {:target_value, :off | :previous | 0..99} | {:duration, 0..255}
+  @type param :: {:target_value, :off | :previous | 0..99} | {:duration, 0..7560 | :default}
```

Comment:
```suggestion
  @type param :: {:target_value, :off | :previous | 0..99} | {:duration, 0..7620 | :default}
```
From other code, it looked like the max value was 7620. I think this is another vote for just capping the max duration when encoding to avoid accidentally raising when the longest duration is probably ok.

Reply:
Fixed. I defined a `duration` type in SwitchSupport to enforce consistency. I also fixed the `param` type for SwitchBinary.